web/src/engine/websites/PiccomaFR.ts

```diff
@@ -0,0 +1,116 @@
+import { type ImageLinks, type TileGroup, script } from './Piccoma';
+import { Tags } from '../Tags';
+import icon from './Piccoma.webp';
+import { DecoratableMangaScraper, type MangaPlugin, Manga, Chapter, Page } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchJSON, FetchRequest, FetchWindowScript } from '../FetchProvider';
+import DeScramble from '../transformers/ImageDescrambler';
+import type { Priority } from '../taskpool/TaskPool';
+
+type APIMangas = {
+    data: {
+        p_products: [{
+            product_id: number,
+            title: string,
+        }]
+    }
+}
+
+type NextData = {
+    props: {
+        pageProps: {
+            initialState: {
+                episode: {
+                    episodeList: {
+                        episode_list?: NextChapters
+                    }
+                },
+            }
+        }
+    }
+}
+
+type NextChapters = {
+    id: string,
+    title: string,
+    volume: number,
+    is_free: 'Y' | 'N',
```

Comment:
Just being picky for the sake of being picky but i think is_free isnt used anywhere.

Reply:
Removed in 7edd06cc